incubator/crate/values.yaml

```diff
@@ -0,0 +1,54 @@
+# The Chart for a crateDB deployment with working defaults consists of:
+#   1) A Service (exposing the ports that crate uses)
+#   2) A StatefulSet (initialising the cluster)
+#   3) Resources definition
+app: crate
+
+image:
+  name: crate
+  tag: 3.1.1
+  pullPolicy: IfNotPresent
+
+# 1) The Service connecting 3 ports.
+service:
+  name: crate
+  type: ClusterIP
```

Comment:
Any reason why you chose a ClusterIP and not a load balancer for the default service type?

Reply:
Well, mainly because ClusterIP is k8s default and since we use crate as our backend, we typically don't expose it to an external IP.